frontend/src/Components/CareersForm/CareersForm.jsx

```diff
@@ -35,132 +40,201 @@ const CareersForm = () => {
       body: yupstring()
         .required(t("formValidation.required.body"))
         .max(500, t("formValidation.length.body")),
-      resumeText: yupstring().when("format", {
+      resumeText: yupstring().when("resumeFormat", {
         is: "paste",
         then: yupstring()
           .required(t("formValidation.required.resumeText"))
-          .max(1000, t("formValidation.length.resumeText"))
-      })
-    })
+          .max(1000, t("formValidation.length.resumeText")),
+      }),
+    }),
   });
   const [radioValue, setRadioValue] = useState("upload");
   const [refreshValue, setRefreshValue] = useState(0);
-  const updateRadio = e => {
+  const [snackbar, setSnackbar] = useState({
+    severity: "error",
+    message: "",
+  });
+  const [openSnackbar, setOpenSnackbar] = useState(false);
+  const updateRadio = (e) => {
     setRadioValue(e.target.value);
   };
 
-  const onFile = fileArr => {
-    sendFile(fileArr[0]);
+  //called when a file is dropped
+  const onDrop = (file) => {
+    sendFile(file)
+      .then((response) => {
+        if (response.status === 200) {
+          setOpenSnackbar(true);
+          setSnackbar({
+            severity: "success",
+            message: t("careers.form.success"),
+          });
+        }
+      })
+      .catch((error) => {
+        if (error.response.status === 500) {
+          setOpenSnackbar(true);
+          setSnackbar({
+            severity: "success",
```

Comment:
I think this somehow missed the commit, but it should be `severity: "error"` yeah?

Reply:
I swear I changed this...

Reply:
Fixed.